### ISSUE TITLE
Lab 06 Issue with NMTC Data

### ISSUE THREAD

**Issue by lindaalvarez** [Watts-College/paf-515-template#21]
Hello @castower, I am following along the Lab 06 code through video and came across an error when trying to load the data, attached is a screenshot of the error message. I am not sure what the issue could be as I have not renamed my divisional data. 

Thank you for your time and help

![Screenshot 2024-11-26 155429](https://github.com/user-attachments/assets/603ec515-4bcb-4b69-9958-6e1dbe9c8bf2)

**Comment by castower** [comment]
Hi @lindaalvarez ,
Are you able to see the files in your data/rodeo folder?

**Comment by lindaalvarez** [comment]
> Hi @lindaalvarez , Are you able to see the files in your data/rodeo folder?

Yes, I am able to open both of the files. I can go ahead and email you my files in case that is a bit easier to look into.

**Comment by castower** [comment]
Yes, that'd be helpful @lindaalvarez

**Comment by castower** [comment]
Hi @lindaalvarez ,
I took a look at your files and your file paths look correct.

Are you working inside of your RProject?

Does here::here() return your expected file path?

**Comment by lindaalvarez** [comment]
> here::here()

@castower This is the file path I am getting, should I change it directly to a lab folder?

[1] "C:/Users/linda/OneDrive/Desktop/project-paf-515-2024f-team-08"
